source/mpi_shared_linear_elasticity.cpp

```diff
@@ -480,8 +486,23 @@ namespace Solid
                           quad_stress[i][j][q] = tmp_stress[i][j];
                         }
                     }
+                  tmp_cell_stress[0] += (0.25 * quad_stress[0][0][q]);
+                  tmp_cell_stress[1] += (0.25 * quad_stress[0][1][q]);
+                  tmp_cell_stress[2] += (0.25 * quad_stress[1][1][q]);
+
+                  if (dim == 3)
+                    {
+                      tmp_cell_stress[3] += (0.25 * quad_stress[0][2][q]);
+                      tmp_cell_stress[4] += (0.25 * quad_stress[1][2][q]);
+                      tmp_cell_stress[5] += (0.25 * quad_stress[2][2][q]);
+                    }
                 }
 
+              for (unsigned int i = 0; i < cellwise_stress.size(); ++i)
+                {
+                  cellwise_stress[i][cell->active_cell_index()] =
+                    tmp_cell_stress[i];
+                }
```

Comment:
I don't think this works in parallel. Please doublecheck. I expect only rank 0 to have correct results and other subdomains will output 0 all the time.